docs/docs/mdx/getting-started.md

```diff
@@ -37,7 +37,7 @@ can follow these steps for configuring the [gatsby-mdx](/packages/gatsby-mdx/) p
 1. **Add `gatsby-mdx`** and MDX as dependencies
 
    ```sh
-   yarn add gatsby-mdx @mdx-js/mdx @mdx-js/tag
+   yarn add gatsby-mdx @mdx-js/mdx @mdx-js/react
```

Comment:
Should we add the migrating link here somewhere?

Reply:
Good call, just pushed a change to add the migration link.